pkg/cloud/amazon/s3_storage.go

```diff
@@ -710,6 +710,11 @@ func (s *s3Storage) newClient(ctx context.Context) (s3Client, string, error) {
 		if s.opts.usePathStyle {
 			options.UsePathStyle = true
 		}
+
+		if s.opts.skipChecksum {
+			// Try to skip checksums when it isn't required.
+			options.RequestChecksumCalculation = aws.RequestChecksumCalculationWhenRequired
```

Comment:
Thank you for the reminder to look at this.

I believe the change in this PR is a no-op. We set `cfg.RequestChecksumCalculation = aws.RequestChecksumCalculationWhenRequired` on line 649 if the AWS_SKIP_CHECKSUM URI parameter is set (ex BACKUP INTO 's3://...&AWS_SKIP_CHECKSUM=true).